options.go

```diff
@@ -251,3 +252,47 @@ type LogOptions struct {
 	// the default From.
 	From plumbing.Hash
 }
+
+var (
+	ErrMissingAuthor    = errors.New("author field is required")
+	ErrMissingCommitter = errors.New("committer field is required")
+)
+
+// CommitOptions describes how a commit operation should be performed.
+type CommitOptions struct {
```

Comment:
I think should be an option (https://git-scm.com/docs/git-commit#git-commit--mltmsggt), but it's depending of how we want to manage the public API.